extensions/amp-sticky-ad/0.1/amp-sticky-ad.css

```diff
@@ -15,4 +15,11 @@
  */
 
  amp-sticky-ad {
+ 	position: fixed;
+ 	bottom: 0;
+ 	left: 0 !important;
+ 	z-index: 1;
+ 	width: 100%;
+ 	display: none;
+ 	max-height: 100px;
```

Comment:
1. The requirement says: the height should be less than 1/6th of the viewport (in portrait mode) or 100 pixels (whichever is smaller). Then I had the discussion with Sriram, he said for current mobile devices the 100px is always the smaller one. 
cc: @sriramkrish85 

2. What I understand is only one `amp-ad` can be put into `amp-sticky-ad`, and the overflow part will be hidden if the ad doesn't fit. 
cc: @jasti